### PR TITLE
Adds the Snow Golem

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

this pr adds snow golems, they are made by using 10 sheets of snow on a golem shell
snow golems have slightly less armor than regular golems and get triple burn damage
they are immune to snowstorms, can create snowballs and have cryokinesis powers
on death they turn into a pile of snow and a carrot
now with the species license (thank you cobby)
image

## Why It's Good For The Game

more golem types are always fun
allows some possible interaction between xenobio and genetics
snow gang

## Changelog
:cl:
add: You can now create snow golems! They have ice powers, but take more damage from burns, heat and fire.
/:cl:

### CODE REVIEW

**Comment by knacker48** [comment]
Oh hey I made an idea thread for this. Good to see someone actually making this

**Comment by nemvar** [comment]
Lame, no carrot nose or top hat.

**Comment by Fikou** [comment]
> Lame, no carrot nose or top hat.

i would be happy to change it if there was a way to change only 1-2 limb sprites but you have to change them all i think

**Comment by Fikou** [comment]
> you can use the `random_name` proc to give them names like "Winter" "Frosty" and other stuff.

meh special names already exists for that and only bananium and runic golems use random name

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -1045,6 +1045,61 @@
...
 		SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "spooked", /datum/mood_event/spooked)
 
+/datum/species/golem/snow
+	name = "Snow Golem"
+	id = "snow golem"
+	limbs_id = "sn_golem"
+	fixed_mut_color = "null" //custom sprites
+	armor = 45 //down from 55
+	burnmod = 3 //melts easily
+	info_text = "As a <span class='danger'>Snow Golem</span>, you are extremely vulnerable to burn damage but you can generate snowballs and shoot cryokinetic beams. You will also turn to snow when dying, preventing any form of recovery."
```
[the remark this answers is not in the document]
Comment (Fikou) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: ok

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -1045,6 +1045,61 @@
...
+	desc = "Concentrates cryokinetic forces to create snowballs, useful for throwing at people."
+	item_type = /obj/item/toy/snowball
+	charge_max = 15
+	action_icon = 'icons/obj/toy.dmi'
+	action_icon_state = "snowball"
+	
+/obj/effect/proc_holder/spell/targeted/conjure_item/snowball/cast(list/targets, mob/user = usr)
+	..()
+	for(var/mob/living/carbon/C in targets)
+		C.throw_mode_on()
```
[the remark this answers is not in the document]
Comment (Fikou) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: a

**Comment by Fikou** [comment]
image
he comin for you in those boxing gloves

**Comment by nicbn** [comment]
It'd be neat to have them dissolve into a tile of water if temperature is too high (maybe make them require subzero temperatures but give them some abilities)

**Comment by PKPenguin321** [comment]
>no jack-o-lantern head

**Pull request merged**